content/tutorials/quickstart/create-accounts-send-xrp.md

```diff
@@ -243,7 +228,7 @@ Otherwise, populate the operational account fields.
 ```
 
 
-Insert the seed values for both accounts as they are created to the **Seeds** field as a convenience. You can copy the values and store them offline, then paste them into the **Seeds** field to retrieve the accounts with the `getAccountsFromSeeds()` function.
+Insert the seed values for both accounts as they are created to the **Seeds** field as a convenience. You can copy the values and store them offline. when you reload this form or another in this tutorial, copy and paste them into the **Seeds** field to retrieve the accounts with the `getAccountsFromSeeds()` function.
```

Comment:
Looks like a missing capital letter at least. Not sure if there's a bigger typo going on here.

```suggestion
Insert the seed values for both accounts as they are created to the **Seeds** field as a convenience. You can copy the values and store them offline. When you reload this form or another in this tutorial, copy and paste them into the **Seeds** field to retrieve the accounts with the `getAccountsFromSeeds()` function.
```